src/ctelnet.cpp

```diff
@@ -1933,14 +1933,8 @@ void cTelnet::setATCPVariables(const QByteArray& msg)
 
 void cTelnet::setGMCPVariables(const QByteArray& msg)
 {
-    QString transcodedMsg;
-    if (mpOutOfBandDataIncomingCodec) {
-        // Message is encoded
-        transcodedMsg = mpOutOfBandDataIncomingCodec->toUnicode(msg);
-    } else {
-        // Message is in ASCII (though this can handle Utf-8):
-        transcodedMsg = msg;
-    }
+    // JSON is always utf8
```

Comment:
We ought to clarify why this comment is talking about JSON when it is supposed to be doing GMCP stuff!
```suggestion
    // JSON (and thus the GMCP data) is always utf8
```